### PR TITLE
Add myTAI R package

### PR DESCRIPTION
**Pull request by gallardoalba**
Add myTAI R package

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by bgruening** [comment]
The error is: `Error in library("myTAI") : there is no package called ‘myTAI’`. 

Are you sure this is how it is used?

**Comment by gallardoalba** [comment]
Not, sure, but it seems that it is failing when trying to load the library, so it seems that it is not able to install it?

File: recipes/r-mytai/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    - r-rcpparmadillo
+    - r-textshaping
+test:
+  commands:
+    - $R -e "library('myTAI')"  # [not win]
+
+about:
+  home: https://github.com/drostlab/myTAI
+  summary: Investigate the evolution of biological processes by capturing evolutionary signatures in transcriptomes (Drost et al. (2017) <doi:10.1093/bioinformatics/btx835>). The aim of this tool is to provide a transcriptome analysis environment to quantify the average evolutionary age of genes contributing to a transcriptome of interest (Drost et al. (2016) <doi:10.1101/051565>).
+  license: "MIT licence"
```
Comment (gallardoalba) [inline recipes/r-mytai/meta.yaml:76]:
```suggestion
  license: GPLv3
  license_family: GPL
  license_file: LICENSE
```

File: recipes/r-mytai/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+test:
+  commands:
+    - $R -e "library('myTAI')"  # [not win]
+
+about:
+  home: https://github.com/drostlab/myTAI
+  summary: Investigate the evolution of biological processes by capturing evolutionary signatures in transcriptomes (Drost et al. (2017) <doi:10.1093/bioinformatics/btx835>). The aim of this tool is to provide a transcriptome analysis environment to quantify the average evolutionary age of genes contributing to a transcriptome of interest (Drost et al. (2016) <doi:10.1101/051565>).
+  license: GPLv3
+  license_family: GPL
+  license_file: LICENSE
```
Comment (gallardoalba) [inline recipes/r-mytai/meta.yaml:78]:
```suggestion
```

**Comment by gallardoalba** [comment]
Is there more information about that problem? @bgruening

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
And done! Thanks @gallardoalba

**Pull request merged**